pkg/virt-controller/services/multus_annotations.go

```diff
@@ -57,11 +57,13 @@ func (mnap multusNetworkAnnotationPool) toString() (string, error) {
 
 func generateMultusCNIAnnotation(vmi *v1.VirtualMachineInstance) (string, error) {
 	multusNetworkAnnotationPool := multusNetworkAnnotationPool{}
-
-	multusNonDefaultNetworks := multus.FilterMultusNonDefaultNetworks(vmi.Spec.Networks)
-	for i, network := range multusNonDefaultNetworks {
-		multusNetworkAnnotationPool.add(
-			newMultusAnnotationData(vmi, network, fmt.Sprintf("net%d", i+1)))
+	podInterfaceNamer := podinterfacenamescheme.NewPodInterfaceNamer(vmi)
```

Comment:
I think the local var `podInterfaceNamer` should be renamed to `podIfacesNameScheme` or `podIfacesNames`.

Reply:
Done